packages/ui/src/composites/wui-input-text/styles.ts

```diff
@@ -73,11 +73,11 @@ export default css`
   }
 
   wui-icon + .wui-size-md {
-    padding: 15px var(--wui-spacing-m) var(--wui-spacing-l) 42px;
+    padding: 15px var(--wui-spacing-m) var(--wui-spacing-l) 40px;
```

Comment:
`--wui-spacing-3xl` instead `40px` maybe? 

I know it's from designs but 15px is still annoying 🥲